js/ToDoList.js

```diff
@@ -0,0 +1,70 @@
+import { Todoitem } from "./TodoItem .js";
+
+class ToDoList {
+  constructor() {
+    this.tasks = JSON.parse(localStorage.getItem("tasks")) || [];
+    this.renderTasks();
+    // console.log( "this for depug",this.tasks[0]);
+  }
+
+  addTask(taskDescription) {
+    const value = taskDescription.value.trim();
+    const newTask = new Todoitem(value);
+    this.tasks.push(newTask);
+    this.saveIntoLocalStorage();
+    taskDescription.value = "";
+  }
+   
+  removeTask(index) {
+    this.tasks.splice(index, 1);
+    this.saveIntoLocalStorage();
+    this.renderTasks();   
+  }
+
+  renderTasks() {
+    const ullist  = document.getElementById("conteaner-list");
+    ullist .innerHTML = "";
+
+    this.tasks.forEach((e,index) => {
```

Comment:
`task` and not `e` (e is not readable var here as it's not an event to call it e)